corehq/apps/app_manager/views/download.py

```diff
@@ -311,13 +314,17 @@ def download_profile(request, domain, app_id):
     See ApplicationBase.create_profile
 
     """
+    if not request.app.copy_of:
+        make_async_build.delay(request.app)
     return HttpResponse(
         request.app.create_profile()
     )
```

Comment:
we expect the download to finish before the build. the build is so that the next time, the version comparison is against the last build